traitator/tests/test_qctraits.py

```diff
@@ -20,7 +20,7 @@ def _find_nan_rows(table: pd.DataFrame, cols: [str]) -> pd.DataFrame:
 class QCTests(TestCase):
     def setUp(self):
         self.fp_guitar = 'traitator/tests/data/trait_data_n10906.csv'
-        self.fp_tresor = 'data/traitdata_tresor.csv'
+        self.fp_tresor = 'data/Traits_v2.xlsx'
```

Comment:
please keep using the CSV version, since git will be able to show differences. This is not possible for binary formats like xlsx. Also note the correct sorting of rows!

Reply:
my csv file is too big to be uploaded on git. What can i do then? Any idea?